ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1145,7 +1145,7 @@ router_bgp:
     - "< bgp command as string >"
   peer_groups:
     < peer_group_name_1>:
-      type: < ipv4 | evpn >
+      type: < ipv4 | evpn | rtc >
```

Comment:
The type knob `rtc` doesn't appear to be used.

Reply:
That's if someone wants to use a different peer-group other than evpn, in our case EVPN does the job as we peer to the same Route Servers or Spines. For flexibility if you want to have different peer-group used only for RTC

Reply:
- [ ] Remove type from readme

Reply:
If we remove the type then then there is no way to describe which AF the peer-group belongs for documentation